.travis.yml

```diff
@@ -10,10 +10,8 @@ before_install:
 - git checkout -
 script:
 - COVERAGE=y npm test
-# Run changelog-lint but only on newer versions of Node (because of syntax errors)
-- if [[ ${TRAVIS_NODE_VERSION:0:1} -ge "4" ]]; then
-    npm run changelog-lint;
-  fi
+- npm run changelog-lint;
```

Comment:
oops, sorry, I didn't notice the semi-colon. Can you remove that? It's not needed since there's no longer an if statement